packages/components/package.json

```diff
@@ -24,9 +23,9 @@
     "linkDirectory": true
   },
   "scripts": {
-    "dev": "stencil build --serve --port 9200 --watch --docs-readme",
-    "start": "stencil build --watch --docs-readme",
-    "build": "stencil build --docs-readme",
+    "dev": "stencil build --port 9200 --serve --watch",
```

Comment:
If you remove docs-readme and push a new/changed component without building first, you'll have changes on the CI/CD that will not be committed.

Reply:
I'will revert that.